### DISCUSSION TITLE
How to use ffmpeg to stream webRTC through whep?

### DISCUSSION THREAD

**Discussion by laixiao2001** [bluenviron/mediamtx#3745] [Questions]
### Question

I manually compiled FFMPEG and support whep streaming. I have successfully streamed it on SRS, but I couldn't find an example of how to stream it on mediaMTX.Does anyone know how to do it? I am extremely grateful.

**Comment by aler9** [comment]
Hello, WHIP/WHEP support in FFmpeg is currently present only in an [unofficial fork](https://github.com/ossrs/ffmpeg-webrtc) maintained by the SRS team. There's no such support in mainline FFmpeg.

We cannot provide help on something that is not official and might suddenly change. If you want to publish with WebRTC, use one of the clients listed in the README. You can also ask the FFmpeg team to implement WHIP.
